lib/turn.rb

```diff
@@ -0,0 +1,36 @@
+class Turn
+  attr_reader :guess, :current_card, :wrong_or_right
```

Comment:
We generally only use `attrs` with instance vars, so `wrong_or_right` should be removed.
```suggestion
  attr_reader :guess, :current_card
```